docs/algorand-art-mint-SC.drawio

```diff
@@ -0,0 +1,258 @@
+<mxfile host="65bd71144e">
```

Comment:
Is this one same as you send me before. if not, could you please attach a screenshot in PR description?

Reply:
No it's not. I'll try to add. I think there's a better way to insert than screenshot.

Reply:
it can't render the 20KB SVG. Uploaded a PNG at last